google/resource_compute_region_autoscaler_test.go

```diff
@@ -0,0 +1,262 @@
+package google
```

Comment:
Can you also add import acceptance tests?

By convention, we put them in a separate file. See `import_compute_autoscaler_test.go"